kie-wb-parent/kie-wb-home-page-community/src/main/java/org/kie/workbench/client/home/HomeProducer.java

```diff
@@ -45,30 +55,62 @@ public HomeModel get() {
+                                                            () -> placeManager.goTo(SERVER_MANAGEMENT),
+                                                            SERVER_MANAGEMENT,
+                                                            PERSPECTIVE);
+        devOps.addLink(new HomeShortcutLink(translationService.format(Constants.Deployments),
+                                            DEPLOYMENTS));
+        devOps.addLink(new HomeShortcutLink(translationService.format(Constants.Servers),
+                                            SERVER_MANAGEMENT));
+
+        final HomeShortcut manage = ModelUtils.makeShortcut("fa-briefcase",
+                                                            translationService.format(Constants.Manage),
+                                                            translationService.format(Constants.ManageDescription),
+                                                            () -> placeManager.goTo(PROCESS_INSTANCES),
+                                                            PROCESS_INSTANCES,
+                                                            PERSPECTIVE);
+        manage.addLink(new HomeShortcutLink(translationService.format(Constants.ProcessDefinitions),
+                                            PROCESS_DEFINITIONS));
+        manage.addLink(new HomeShortcutLink(translationService.format(Constants.ProcessInstances),
+                                            PROCESS_INSTANCES));
+        manage.addLink(new HomeShortcutLink(translationService.format(Constants.TasksAdmin),
+                                            TASKS_ADMIN));
+        manage.addLink(new HomeShortcutLink(translationService.format(Constants.Jobs),
+                                            JOBS));
+        manage.addLink(new HomeShortcutLink(translationService.format(Constants.ExecutionErrors),
+                                            EXECUTION_ERRORS));
+
+        final HomeShortcut track = ModelUtils.makeShortcut("pficon-trend-up",
+                                                           translationService.format(Constants.Track),
+                                                           translationService.format(Constants.TrackDescription),
+                                                           () -> placeManager.goTo(APPS),
+                                                           APPS,
+                                                           PERSPECTIVE);
+        track.addLink(new HomeShortcutLink(translationService.format(Constants.Tasks),
+                                           TASKS));
+        track.addLink(new HomeShortcutLink(translationService.format(Constants.Reports),
+                                           PROCESS_DASHBOARD));
+        track.addLink(new HomeShortcutLink(translationService.format(Constants.BusinessDashboards),
+                                           APPS));
```

Comment:
@paulovmr I'm wondering if instead of APPS, this should be PROCESS_DASHBOARD and also used as the default action for the Track link.